app.js

```diff
@@ -0,0 +1,468 @@
+      console.log('\nDetailed results per file:');
+      console.dir(fileResults, { depth: null });
+
+      console.log('\nFiles per STIX Malware Analysis result:');
+      const resultValues = [
+        ...new Set([
+          ...STIX_MALWARE_RESULT_OV,
+          ...fileResults
+            .map((fileResult) => fileResult.stixMalwareAnalysis.result)
+            .sort(),
+        ]),
+      ];
+      resultValues.map((resultValue) => {
+        console.log('- ' + resultValue + ' :');
+        console.log(
+          fileResults
+            .filter(
+              (fileResult) =>
+                fileResult.stixMalwareAnalysis.result === resultValue,
+            )
+            .map((fileResult) => fileResult.file),
+        );
+      });
+
+      // TODO: Let storeMalwareAnalysis() run a SELECT query after the insert
+      //       to check in which (if any) graphs the resource was inserted?
+      //       Would be more reliable and easier to parse.
+      const filesNoDatabaseUpdate = fileResults.filter(
+        (fileResult) =>
+          !(
+            fileResult.databaseResponse?.results?.bindings[0]?.['callret-0']
+              ?.value &&
+            fileResult.databaseResponse?.results?.bindings[0]?.[
+              'callret-0'
+            ]?.value.match(' -- done')
+          ),
+      );
+      if (filesNoDatabaseUpdate.length) {
+        console.log(
+          '\nFiles for which the database response indicates that the ' +
+            'malware analysis resource object was not added to any graph:',
+        );
+        console.dir(filesNoDatabaseUpdate, { depth: null });
+      }
+    } catch (e) {
+      console.log(e);
+      res.status(500).send('Uncaught error in /delta: ' + e);
+      // TODO: Re-throw error? Because not sure if the response 500
+      //       will be received (e.g. /delta already sent 202 or /post
+      //       connection timeout during long scan). If the result was
+      //       not "benign", but not stored because of an error, an
+      //       earlier "benign" result for the same file will remain the
+      //       latest result. Perhaps such errors should require more
+      //       attention.
+      //       OTOH: - Re-throwing lets the entire service crash.
+      //       - Perhaps it is the user's responsibility anyway to
+      //       check that the last malware-analyse is recent enough.
+      //       - Already a note about this in README.
+      //       Could be somewhat mitigated by storing a malware analysis
+      //       with only analysis-started before the scan, and update it
+      //       after the scan.
+    }
+  },
+);
+
+/**
+ * Scans a single file and stores the result.
+ *
+ * @param {Object} body Request body should be in JSON-format with
+ *                      `file` containing a logical file IRI as a single String.
+ *                      E.g. { "file": "http://mu.semte.ch/services/file-service/files/6543bc046ea4f3000e00000c" }
```

Comment:
Before making this API public we should see how json:api-compatible it can be made. Again not for this iteration. Very good it's there